services/frontend-service/src/ui/components/Spinner/Spinner.scss

```diff
@@ -25,5 +25,13 @@ Copyright 2023 freiheit.com*/
     }
     .spinner-message {
         margin-top: 20px;
+        font-size: 18px;
```

Comment:
I understand we don't follow the figma design anymore but it would be nice to still use the global typography definitions for a cleaner look

Reply:
done